sros2/sros2/api/_policy.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Canonical Ltd
+# Copyright 2016-2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import lxml
```

Comment:
Any reason for having changed `from lxml import etree` -> `from lxml` ?

Reply:
Because multiple packages provide etree. By using `lxml` it's immediately clear which one we're using and a little harder to break in future refactors.

Reply:
Make sense. You could also do something like `from lxml import etree as ltree` for shortening a bit altho it's not really necessary here.

Reply:
I've removed this. Not trying to actually refactor, here, my apologies.

Reply:
I'm fine with being explicit to avoid ambiguity and improve future refactors. If reverted here, please also revert it in the `_keystore` module that had the same change so that we have the same state everywhere

Reply:
I'm not actually intending to refactor beyond what I need to do to enable the reorg, the keystore shouldn't have had it either. I'll make a follow-up.

Reply:
sgtm